vhpidirect/cinterface/accarraygen/caux.c

```diff
@@ -0,0 +1,11 @@
+#include <stdio.h>
+#define SIZE_ARRAY (sizeof(intArray)/sizeof(int))
+
+int intArray[5];
```

Comment:
I think that this won't work, because the generic must be `<=5`. I suggest to use `int *intArrayDB` instead. Then, `getIntArr_ptr` needs to receive `SIZE_ARRAY` as an argument.

Reply:
Yeah... That was an eye-sore. Will ammend